code/modules/awaymissions/mission_code/wildwest.dm

```diff
@@ -80,19 +80,40 @@
 				user.mutations.Add(LASER)
 				user.mutations.Add(RESIST_COLD)
 				user.mutations.Add(XRAY)
-				user.dna.mutantrace = "shadow"
+				if(ishuman(user))
+					var/mob/living/carbon/human/human = user
+					if(human.species.name != "Shadow")
```

Comment:
This should probably use H for consistency, and if the human doesn't have a species datum, it'll throw runtimes. Also not sure why the wishgranter would even set people to being a shadow.